Document system welcome messages for CLAN servers (#273)
discord-userdoccers/discord-userdoccers: pages/resources/message.mdx

> The type of rendered message is determined via converting the message's `timestamp` to a unix timestamp with millisecond precision, modulo 13.

Reviewer: ```suggestion
 Usually, the type of rendered message is determined via converting the message's `timestamp` to a unix timestamp with millisecond precision, modulo 13.
```

> However, if the guild has `CLAN` feature, the same is determined via dividing the message's `timestamp` with millisecond precision by modulo 10.
> 
> See the 2nd table below for system welcome messages that shows up in CLAN servers.

Reviewer: ```suggestion
However, if the guild has the [`CLAN` feature](/resources/guild#guild-features), the same is determined via dividing the message's `timestamp` with millisecond precision by modulo 10.

See the 2nd table below for the system welcome messages specific to clans.
```

| 11    | "Good to see you, {author}."                    |
| 12    | "Yay you made it, {author}!"                    |

> 

Author: what should the new heading be called

> 

Author: if we add new heading, this section in https://docs.discord.sex/resources/message#message-type will also require new jump link for clarification
![image](https://github.com/user-attachments/assets/ff008e39-f904-49d8-bf1a-c905edea81ac)

> 

Reviewer: Yup, moving the new clan welcome message description in between the tables would be perfect

> 

Author: ok done, how does it look now :3
